CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.2.9] - 2023-06-20
```

Comment:
since you're removing an argument, this is a very breaking change for users.
Could we release v 0.3.0?

Reply:
Can do

Reply:
Should we keep the argument and deprecate it so that we don't have to bump the minor ?

Reply:
This would be the smoothest transition.
Keep them in the `create_batch` and `get_batch` functions, mark as deprecated and ignore.

Reply:
OK